### PR TITLE
data-interchange

### PR DESCRIPTION
**Pull request by ghost**
Added new route for images not coming from image_tag()

The images are all the same one right now for interchange but they can
be changed later. I did test with different images to make sure this
worked

### CODE REVIEW

**Comment by ghost** [comment]
For anyone changing the settings on interchange:
screen shot 2016-08-13 at 3 55 43 pm

File: config/routes.rb [hunk 1]
```diff
@@ -3,5 +3,6 @@
 
   resource :home, controller: :home
   resources :events, only: [:index, :show]
+  get ':name.:ext', to: redirect('/assets/%{name}.%{ext}'), constraints: { name: /.+/, ext: /(jpg|png|gif)/}
```
Comment (mbyczkowski) [inline config/routes.rb]: I'd probably avoid creating separate routes for images by having proper paths set-up in HTML.

File: app/views/pages/volunteer.html.haml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .row
-  = image_tag 'home_banner_1_placeholder.png'
+  = image_tag "","data-interchange" => "[home_banner_1_placeholder.png, large],[home_banner_1_placeholder.png, only screen and (min-width: 40.063em) and (max-width: 64em)],[home_banner_1_placeholder.png, only screen and (max-width: 40em)]"
```
Comment (mbyczkowski) [inline app/views/pages/volunteer.html.haml]: We should be using `image_path` (a Rails helper) to generate proper image paths. Here's a good example with interchange: https://coderwall.com/p/uip7xg/ruby-on-rails-image-helper-for-retina-display-with-foundation-interchange.
In that example they used a helper to create a retina image, but we could have a helper that automatically generates `_small`, `_medium` and `large` image paths, and then we'd have to ensure that we always have images for all 3 sizes.

**Comment by mbyczkowski** [comment]
@mhgump apart frome some nit-picks (inline comments), I think this is the right way to go about interchange.

**Comment by ghost** [comment]
I've added some more commits that I think should fix those issues

**Comment by mbyczkowski** [comment]
cool, let's go with that 👍

**Pull request merged**